tests/step_implementers/deploy/test_argocd.py

```diff
@@ -368,7 +369,8 @@ def test_ArgoCD_run_step_success(
             )
             argocd_app_sync_mock.assert_called_once_with(
                 argocd_app_name='test-app-name',
-                argocd_sync_timeout_seconds=60
+                argocd_sync_timeout_seconds=60,
+                argocd_sync_retry_limit=3
```

Comment:
Can you rebase against `main` before merge? The last PR merge added a new test that will be missing this param -- I checked against my local, that test will fail after merge if nothing is done.